packages/common/src/initiatives/HubInitiative.ts

```diff
@@ -253,11 +254,22 @@ export class HubInitiative
    * @param editorContext
    * @returns
    */
-  toEditor(editorContext: IEntityEditorContext = {}): IHubInitiativeEditor {
-    // Cast the entity to it's editor
-    const editor = cloneObject(this.entity) as IHubInitiativeEditor;
+  async toEditor(
+    editorContext: IEntityEditorContext = {},
+    include: string[] = []
+  ): Promise<IHubInitiativeEditor> {
+    // 1. optionally enrich entity and cast to editor
+    const editor = include.length
```

Comment:
I was sort of expecting to see each entity type provide it's own list of default `include`s. So the default for initiatives (or sites) would be `['followersGroup']`. That way the default behavior of `initiativeEntity.toEditor()` would be to include the followers group information. You could override that default behavior by passing `[]` as the `include` argument.

Reply:
Leaving this here for posterity even though we discussed in component standup: rather than having the entities themselves define defaults, defaults are being defined/passed in at the ui level (e.g. from the `arcgis-hub-entity-editor`). This is because the defaults differ depending on the ui-context we're in (not necessarily the entity) - on the followers pane (`editorType = "hub:<entity>:followers"`), we'll always want the `followersGroup` enrichment, but we don't need this on the details pane (`editorType = "hub:<entity>:edit"`)